qemu/tests/balloon_service.py

```diff
@@ -53,7 +53,9 @@ def balloon_memory(vm, mem_check, min_sz, max_sz):
                                                                    tag)
                 time.sleep(20)
                 if mem_check == "yes":
-                    balloon_test.memory_stats_check('stat-free-memory', mem_stat_working)
+                    check_list = ['stat-free-memory', 'stat-disk-caches']
```

Comment:
Is stat-disk-caches supported on rhel7?

Reply:
It depends on driver version for windows guest, virio-win-drive for el7 support it.
Could you check from linux part.

Reply:
Not only from guest side, but also from qemu side.  Had better provide a test result on rhel7.
I don't have a rhel7 host at hand. Maybe you could run a test with a rhel7 guest as well.

Reply:
Let's disable rhel7 guest for stat-disk-caches check. It's not supported for rhel7 kernel.

Reply:
In case we have more check items in the future, how about get check_list from cfg, and add guest restriction in cfg?